File: geotrackr/lib/config/api_config.dart
class ApiConfig {
  //========================== Base Url ==========================//
  static const String baseUrl = 'http://192.168.0.186:8000/api/v1/office';

  //========================== Employee Endpoints ==========================//
  static const String employeeLogin = '$baseUrl/login-employee';

  static const String employeeDetails = '$baseUrl/get-employee-details';

  static const String updateEmployeeDetails =
      '$baseUrl/update-employee-details';

  static const String officeDetails = '$baseUrl/get-office-details';

  //========================== Attendance Endpoints ==========================//
  static const String checkIn = '$baseUrl/employee/check-in';

  static const String checkOut = '$baseUrl/employee/check-out';

  static const String getAttendances = '$baseUrl/employee/attendances';
}

File: dashboard/src/config/api-config.ts
class ApiConfig {
  public static BASE_URL = 'https://geotrackr-server.onrender.com/api/v1';

  public static officeSignup = ApiConfig.BASE_URL + '/office/sign-up';

  public static officeLogin = ApiConfig.BASE_URL + '/office/login';

  public static addOfficeEmployee =
    ApiConfig.BASE_URL + '/office/add-office-employee';

  public static addRemoteEmployee =
    ApiConfig.BASE_URL + '/office/add-remote-employee';

  public static getEmployees = ApiConfig.BASE_URL + '/office/get-employees';

  public static deleteEmployee = ApiConfig.BASE_URL + '/office/delete-employee';

  public static loadOffice = ApiConfig.BASE_URL + '/office/get-details';

  public static updateOffice = ApiConfig.BASE_URL + '/office/update-details';

  public static deleteOffice = ApiConfig.BASE_URL + '/office/delete-office';

  public static addOffice = ApiConfig.BASE_URL + '/office/add-office';

  public static updateOfficeEmployee =
    ApiConfig.BASE_URL + '/office/update-employee';

  public static employeeAttandances =
    ApiConfig.BASE_URL + '/office/employee/attendances';

  public static sendMessage = ApiConfig.BASE_URL + '/contact/send-message';
}

export default ApiConfig;
